OptOutScenarios.tsv

```diff
@@ -0,0 +1,294 @@
+CY-1	ITSA Status								Expected Outcome			dethrfs
+Finalised	CY-1	CY	CY+1	Options presented to customer for opt out within V and C	Customer intent	Valid?		Which years will be sent from V & C to ITSD	CY-1	CY	CY+1	dethrfs
+N	M	M	M	Nothing displayed	Customer wants to opt out from CY-1	N/A		N/A	M	M	M	dethrfs

```

Comment:
When we first did the generation of test data from the sheet, we decided **not** to commit the TSV file, to encourage future versions of the spreadsheet be re-generated from first principles using the helper code in the ignored test (rather than hacking the file). The idea being that this would keep the test scenarios as close as possible to the HMRC spreadsheet (plus it would prevent the helper code from going stale).

I'll let you work out what you want to do with this @chandraAndem.